src/main/java/hudson/plugins/git/GitStatus.java

```diff
@@ -324,7 +324,7 @@ public void writeBody(PrintWriter w) {
         @SuppressFBWarnings(value="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE", justification="Jenkins.getInstance() is not null")
         public List<ResponseContributor> onNotifyCommit(String origin, URIish uri, String sha1, List<ParameterValue> buildParameters, String... branches) {
             if (LOGGER.isLoggable(Level.FINE)) {
-                LOGGER.fine("Received notification from " + StringUtils.defaultIfBlank(origin, "?")
+                LOGGER.fine("Received notification from " + StringUtils.defaultIfEmpty(origin, "?")
```

Comment:
The intent in https://github.com/jenkinsci/git-plugin/pull/497/commits/ce4c2a473ca4209ae23e1703e65cf1b1b9c7506c commit was to make project compile as it didn't for me.
I'll double check and revert it back to `defaultIfBlank` if it works.